components/constellation/constellation.rs

```diff
@@ -760,6 +760,13 @@ where
         join_handle
             .join()
             .expect("Failed to join on the fetch thread in the constellation");
+
+        // Note: the last thing the constellation does,
+        // is asking the embedder to shutdown.
+        // This helps ensure we've shutdown all our internal threads before
+        // de-initializing Servo(see the `thread_count` warning on MacOS).
+        debug!("Asking embedding layer to complete shutdown.");
+        self.embedder_proxy.send(EmbedderMsg::ShutdownComplete);
```

Comment:
Minor nit: "shut down" versus "shutdown" and comment formatting:

```suggestion
        // Note: the last thing the constellation does, is asking the embedder to
        // shut down. This helps ensure we've shut down all our internal threads before
        // de-initializing Servo (see the `thread_count` warning on MacOS).
        debug!("Asking embedding layer to complete shutdown.");
        self.embedder_proxy.send(EmbedderMsg::ShutdownComplete);
```